### PR TITLE
update project_section

### PR DESCRIPTION
**Pull request by vrushalikudande**
updated the index.html, updated the script.js, added images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -418,113 +418,112 @@ <h3 class="h3 card-title">24/7 hours support</h3>
...
-                    <!-- HTML !-->
-                    <button class="button-48" role="button"><span class="text">Check Out</span></button>
-                  </div>
+                <div class="card-content">
+                  <h3 class="h3">
+                    <a href="#" class="card-title">Web Development</a>
+                  </h3>
+                  <!-- <a href="#" class="card-tag">Branding</a> -->
+                  <!-- HTML !-->
+                  <button class="button-48" role="button" onclick="openInNewTab('https://notevid.vercel.app/')"><span class="text">Check Out</span></button>
```
Comment (SAYUK09) [inline index.html:446]: Instead of writing a js function, can't we wrap the `button` in `<a href = " "></a>` directly?

File: assets/js/script.js [hunk 2]
```diff
@@ -76,4 +76,9 @@ carousel.addEventListener('mouseleave', () => {
...
 carousel.addEventListener('mouseenter', () => {
   carousel.classList.add('hover');
-});
\ No newline at end of file
+});
+
+function openInNewTab(url) {
+  var newTab = window.open(url, '_blank');
+  newTab.focus();
+}
```
Comment (SAYUK09) [inline assets/js/script.js:84]: Can't we simply use a `<a>` tag instead of this function?

**Review by SAYUK09 — CHANGES_REQUESTED** [review]
Please address the review comments.
Everything else looks good.

File: assets/js/script.js (hunk 2 above)
Comment (vrushalikudande) [reply assets/js/script.js]: yes.

File: index.html (hunk 1 above)
Comment (vrushalikudande) [reply index.html]: yes.

File: assets/js/script.js (hunk 2 above)
Comment (vrushalikudande) [reply assets/js/script.js]: Changes done.

**Review by SAYUK09 — APPROVED** [review]

**Pull request merged**